web/src/util/index.js

```diff
@@ -97,15 +97,15 @@ export const STANDARDS = [
     title: 'Mitigation Strategy'
   },
   {
-    id: 'key-personnel',
+    id: 'keyPersonnel',
     title: 'Key Personnel'
   },
   {
     id: 'documentation',
     title: 'Documentation'
   },
   {
-    id: 'minimize-cost',
+    id: 'minimizeCost',
```

Comment:
I made these camel case on the expectation that's what they'd be when they come from the API.  Not exactly critically important, but 🤷‍♂️